pkgs/development/tools/misc/chruby-fish/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "chruby-fish";
+
+  version = "0.8.2";
+
+  src = fetchFromGitHub {
+    owner = "JeanMertz";
+    repo = "chruby-fish";
+    rev = "v${version}";
+    sha256 = "15q0ywsn9pcypbpvlq0wb41x4igxm9bsvhg9a05dqw1n437qjhyb";
+  };
+
+  installFlags = [ "PREFIX=$(out)" ];
+
+  meta = {
+    description = "Thin wrapper around chruby to make it work with the Fish shell";
+    homepage = "https://github.com/JeanMertz/chruby-fish";
```

Comment:
Doesn't that means that you should wrap this with chruby?

Reply:
You mean something like installing both chruby and fish automatically enables chruby-fish? I don't know how to achieve that...

With curent implementation, we can at least use chruby-fish without wrapping by

1. `nix-env --install fish chruby chruby-fish`
1. write some config like [here](https://github.com/cohei/dotfiles/blob/cf1d6cb9a20633e4646cbc983e63283d108cedf8/home/.config/fish/config.fish#L29-L31)